Userland/Libraries/LibWeb/XHR/FormData.cpp

```diff
@@ -181,4 +181,9 @@ JS::ThrowCompletionOr<void> FormData::for_each(ForEachCallback callback)
     return {};
 }
 
+ErrorOr<HTML::SerializedFormData> FormData::serialize_to_multipart_form_data()
```

Comment:
https://github.com/SerenityOS/serenity/blob/master/Documentation/Browser/Patterns.md#comments

> When adding non-standard code for a feature that is otherwise well-specified, it should be marked as such.

Reply:
Also, I'm not sure how much value it actually adds. Both one line...

Reply:
Removed this commit and added another one that adds a get accessor to the internal entry list. Also prefixed it with `NON-STANDARD:`